workers/unity/Packages/com.improbable.gdk.health/Systems/HealthRegenSystem.cs

```diff
@@ -146,14 +164,13 @@ protected override void OnUpdate()
                         regenData.NextRegenTimer += regenComponent.RegenInterval;
 
                         // Send command to regen entity.
-                        var commandSender = toRegen.ModifyHealthCommandSenders[i];
                         var modifyHealthRequest = new HealthComponent.ModifyHealth.Request(
                             toRegen.EntityId[i].EntityId,
                             new HealthModifier()
                             {
-                                Amount = regenComponent.RegenAmount
+                                Amount = regenComponent.RegenAmount - 3
```

Comment:
temp hack should be removed

Reply:
This PR is a temporary hack to test performance